Source/OxyPlot/Series/BoxPlotItem.cs

```diff
@@ -81,10 +84,84 @@ public struct BoxPlotItem
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="BoxPlotItem" /> struct.
+        /// </summary>
+        /// <param name="x">The x.</param>
+        /// <param name="dataPoints">Data series with the original values</param>
+        /// <param name="whiskerType">Type of whisker values.</param>
+        public BoxPlotItem(double x,
```

Comment:
I would suggest to move this into the ExampleLibrary as a 'utility' class and use it in an example.